package-lock.json

```diff
@@ -10624,18 +10624,18 @@
       }
     },
     "node_modules/@tanstack/virtual-core": {
-      "version": "3.0.0-beta.54",
-      "resolved": "https://registry.npmjs.org/@tanstack/virtual-core/-/virtual-core-3.0.0-beta.54.tgz",
-      "integrity": "sha512-jtkwqdP2rY2iCCDVAFuaNBH3fiEi29aTn2RhtIoky8DTTiCdc48plpHHreLwmv1PICJ4AJUUESaq3xa8fZH8+g==",
+      "version": "3.0.0-beta.61",
+      "resolved": "https://registry.npmjs.org/@tanstack/virtual-core/-/virtual-core-3.0.0-beta.61.tgz",
+      "integrity": "sha512-C3iu9rkWvjJ/KbYAjuwLTjqiR9fcf9lXjRBLHDzsL/M6cR7/HLSU94abSdVrzuiqRvP7u7e0FeqPK0O3VbvWyg==",
       "funding": {
         "type": "github",
         "url": "https://github.com/sponsors/tannerlinsley"
       }
     },
     "node_modules/@tiptap/core": {
-      "version": "2.1.6",
-      "resolved": "https://registry.npmjs.org/@tiptap/core/-/core-2.1.6.tgz",
-      "integrity": "sha512-gm8n1oiBhSP6CDhalmmWwLD7yzIUqJJ246/t8rY3o+HJ/I+p0rqCx0mPvMiwcIBmYX8tUCVz7mb9aSFUu/umOQ==",
+      "version": "2.1.11",
```

Comment:
@suseendran-ni  could you do whatever additional validation you think is necessary for the tiptap and prosemirror updates? This could include
-  manual testing of the storybook build
-  looking at release notes for breakages or new features we want to use or bug fixes that allow us to remove workarounds

Note: this is just a package-lock update, meaning it affects the versions used by the repo's pipeline but it doesn't affect the versions that clients like SLE are required to install

Reply:
We have manually verified the storybook build and the functionalities in the rich text editor and viewer are working as expected. There are a couple of updates on the [extension-link](https://tiptap.dev/api/marks/link) but we are not using those features right now(might be used when [hyperlink support](https://github.com/ni/nimble/issues/1527) is added). Also, there are no bug fixes that will help us to remove the workarounds.